Helper/AdapterHelper.php

```diff
@@ -4,42 +4,22 @@
 
 use Algolia\AlgoliaSearch\Helper\Adapter\FiltersHelper;
 use Algolia\AlgoliaSearch\Helper\Data as AlgoliaDataHelper;
+use Algolia\AlgoliaSearch\Service\AlgoliaCredentialsManager;
 use Magento\CatalogSearch\Helper\Data as CatalogSearchDataHelper;
 
 class AdapterHelper
 {
     public const INSTANTSEARCH_ORDER_PARAM = 'sortBy';
     public const BACKEND_ORDER_PARAM = 'product_list_order';
 
-    /** @var CatalogSearchDataHelper */
-    private $catalogSearchHelper;
-
-    /** @var AlgoliaDataHelper */
-    private $algoliaHelper;
-
-    /** @var FiltersHelper */
-    private $filtersHelper;
-
-    /** @var ConfigHelper */
-    private $configHelper;
-
-    /**
-     * @param CatalogSearchDataHelper $catalogSearchHelper
-     * @param AlgoliaDataHelper $algoliaHelper
-     * @param FiltersHelper $filtersHelper
-     * @param ConfigHelper $configHelper
-     */
```

Comment:
Just curious, why did we remove the doc blocks from constructors?

Reply:
It was made to be consistent with the latest added classes which have no blocks, I also like to remove lines as it doesn't really bring any value in this case.